deployments/operator/samples/deviceplugin_v1_qatdeviceplugin.yaml

```diff
@@ -2,12 +2,11 @@ apiVersion: deviceplugin.intel.com/v1
 kind: QatDevicePlugin
 metadata:
   name: qatdeviceplugin-sample
-  # example apparmor annotation
-  # see more details here:
+  annotations:
```

Comment:
The changes in this file can also be dropped

Reply:
@mythi
This can't be dropped, the operator overrides plugin's annotations.

Reply:
@ozhuraki right, true, I forgot that. Perhaps the operator override logic could be updated so that these do not have to be enabled here

Reply:
@ozhuraki let's get this finished and update the controller logic to only allow updates to annotation keys already set in the daemonset instead of deepcopy.